src/platform/input/constants.js

```diff
@@ -52,6 +52,20 @@ export const EVENT_MOUSEUP = 'mouseup';
  */
 export const EVENT_MOUSEWHEEL = 'mousewheel';
 
+/**
+ * Name of event fired when the mouse moves out or enters another DOM element.
```

Comment:
```suggestion
 * Name of event fired when the mouse moves out or enters another DOM element. This can fire on entering a defocused PlayCanvas browser window so you may want to check if it is focused with {@link https://developer.mozilla.org/en-US/docs/Web/API/Document/hasFocus document.hasFocus()}
```

Reply:
maybe 'background' is a better term?

Reply:
Updated suggested change with something with less jargon

Reply:
Just changed it via normal commit, GitHub is having some strange issues:

![image](https://github.com/playcanvas/engine/assets/5236548/29750efb-9e77-4943-863f-ab12fef65c1d)

(for my sake it sounds good enough, but you can make a suggestion if needed)